.github/workflows/e2e.yml

```diff
@@ -1,8 +1,8 @@
 name: e2e tests
 
 on: 
-  push:
-    branches: [main, vnext]
+  # push:
+  #   branches: [main, vnext, k8s-sync-* ] # TODO: k8s-sync-* value back
```

Comment:
Why do we comment out these branches?

Reply:
Because the demo PR is also triggered, I don't want this action to be triggered as well, so removed temporarily, will change the value back.